app/models/manageiq/providers/kubernetes/container_manager_mixin.rb

```diff
@@ -124,31 +124,31 @@ def default_authentication_type
   end
 
   def scan_job_create(entity)
-    check_policy_prevent(:request_containerimage_scan, entity, :raw_scan_job_create, entity)
+    check_policy_prevent(:request_containerimage_scan, entity, :raw_scan_job_create)
   end
 
-  def raw_scan_job_create(entity)
+  def raw_scan_job_create(class_name, id)
```

Comment:
What did the separation here buy you?

I understand that objects don't want to go into the `MiqQueue` or `Job.create_job`. I don't see any invocations to `:raw_scan_job_create` going onto the stack